pkg/cmd/gtctl/connect/mysql/mysql.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mysql
+
+import (
+	"errors"
+	"github.com/spf13/cobra"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+const (
+	UpSeparator    = ":"
+	AtSeparator    = "@"
+	ArgHost        = "-h"
+	ArgPort        = "-P"
+	ArgUser        = "-u"
+	ArgPassword    = "-p"
+	MySQL          = "mysql"
+	SplitSeparator = "://"
+)
+
+func ConnectCommand(cmd *cobra.Command, args []string) error {
```

Comment:
suggestion: I think it's too complicated for the user to use `connect`. Can we use the protocol argument, for example:

```
gtctl cluster connect <cluster-name> -p mysql

gtctl cluster connect <cluster-name> -p pg
```

We can get the cluster meta info from `get cluster`.